### PR TITLE
hw4

### PR DESCRIPTION
**Pull request by dima666Sik**
pull_hw4

### CODE REVIEW

File: hw4_zd1.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let sumValue = {
+    a:3,
+    b:7
+};
+
+function calc() {
+    return this.a + this.b;
+}
+
+let funcSum = calc.bind(sumValue);
```
Comment (StanislawVictorovich) [inline hw4_zd1.js]: Make execution without wrapper please

File: hw4_zd2.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+function User(firstName, lastName, age) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+    this.print = function () {
```
Comment (StanislawVictorovich) [inline hw4_zd2.js]: This is not a correct and not optimized way of adding method.

File: hw4_zd1.js (hunk 1 above)
Comment (dima666Sik) [reply hw4_zd1.js]: Done

File: hw4_zd2.js (hunk 2 above)
Comment (dima666Sik) [reply hw4_zd2.js]: Done

**Review by StanislawVictorovich — APPROVED** [review]

**Pull request merged**